test/e2e/README.md

```diff
@@ -191,6 +191,43 @@ You can load the new image into the kind cluster using the command below:
 ```bash
 kind load docker-image projects.registry.vmware.com/antrea/antrea-ubuntu:latest --name <kind_cluster_name>
 ```
+## Running the e2e tests on existing k8s cluster
+If you already have a k8s cluster and push antrea on it , the following steps should be done for running the e2e tests or other kinds of tests.
+
+First , the configuration of the k8s cluster is detected and used when running the tests , so the `config` file of the cluster should be copied to the certain place:
+```bash
+cd <directory containing this README file>/infra/vagrant/playbook
+mkdir -p kube
+sudo cp -i /etc/kubernetes/admin.conf kube/config
```

Comment:
I don't think you can assume all existing K8s clusters has this file, and one question about `kube`, should it be `.kube` or `kube`?